src/components/app/App.js

```diff
@@ -87,37 +88,37 @@ class App extends Component {
   };
 
   render() {
-    // todo 4) достать также в строке 92 из стейта selectedOption
-    const { posts } = this.state;
+    // done_todo 4) достать также в строке 92 из стейта selectedOption
+    const { posts, selectedOption } = this.state;
 
     return (
       <div className="App">
         <Header />
 
-        <PanelFromLecture isOpenByDefault={false} >
+        <PanelFromLecture label="Hello World Label" isOpenByDefault={false} >
           Hello, world!
         </PanelFromLecture>
 
         <PanelFromLecture label="test">
           One more panel
         </PanelFromLecture>
 
-        <PanelFromLecture label="Posts">
+        <PanelFromLecture label="Posts" isOpenByDefault={true}>
```

Comment:
Can be just `isOpenByDefault`  without `={true}`
It's not a mistake, it's just a feature of booleans in jsx